<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileReader</class>
 <widget class="QWidget" name="FileReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MQ Listener</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="boxConnection">
         <property name="title">
          <string>File reader</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="verticalSpacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblConnection">
            <property name="text">
             <string>File to open</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLineEdit" name="editConnection"/>
            </item>
            <item>
             <widget class="QToolButton" name="btnConnection">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblInput">
            <property name="text">
             <string>Input</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="rbBinary">
              <property name="text">
               <string>&amp;Binary</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbText">
              <property name="text">
               <string>&amp;Text</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QComboBox" name="cmbCodec"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblReadMode">
            <property name="text">
             <string>Read Mode</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cmbReadMode"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblBytes">
            <property name="text">
             <string>Length</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="spinBytes">
            <property name="suffix">
             <string> bytes</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1024</number>
            </property>
            <property name="value">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblFileFormat">
            <property name="text">
             <string>File Format</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="cmbFileFormat"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lblInterval">
            <property name="text">
             <string>Interval</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="spinInterval">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="minimum">
             <number>500</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>500</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QCheckBox" name="chkRotate">
            <property name="text">
             <string>Rotate</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnConnect">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDisconnect">
              <property name="text">
               <string>Disconnect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxDecoder">
         <property name="title">
          <string>Decoder</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutDecoder">
          <item>
           <widget class="QComboBox" name="cmbDecoder"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxReply">
         <property name="title">
          <string>Reply</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="cmbReplyType">
            <item>
             <property name="text">
              <string>No Reply</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Echo Reply</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Predefined Text</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Predefined Binary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Defined by Action</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editReply"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxAction">
         <property name="title">
          <string>Action</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="cmbHandler"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>347</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="textLog"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
